### PR TITLE
add issue templates

### PR DESCRIPTION
**Pull request by MikeVedsted**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
```
Comment (Rostislav013) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]: Shall we add actual result?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
+ - Version [e.g. 22]
```
Comment (Rostislav013) [inline .github/ISSUE_TEMPLATE/bug_report.md:35]: How about to combine it?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (MarinaFroes) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]: Maybe adding a current behavior session would be good. We could either keep the describe session as a summary and this one as a detailed explanation or  we could just start with "Expected", then "Current (or Actual)" and the steps to reproduce it.

**Current behavior**
A clear and concise description of what is actually happening.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (Rostislav013) [reply .github/ISSUE_TEMPLATE/bug_report.md:35]: Example:
```
Device/os: [Samsung S10, Andoroid 7]
Browser/version: [Chrome, 55]
```

Comment (MarinaFroes) [inline .github/ISSUE_TEMPLATE/bug_report.md:35]: We could have just an Environment session and merge those 2?


**Environment**
- Device (if mobile): [e.g. iPhone6]
 - OS: [e.g. iOS8.1]
 - Browser [e.g. chrome, safari]
 - Version [e.g. 22]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+**Additional context**
+Add any other context about the problem here.
```
Comment (MarinaFroes) [inline .github/ISSUE_TEMPLATE/bug_report.md:38]: After this, I think it's nice to have a session for suggesting solution or even saying what the person tried already.

**Possible solution**
If applicable, describe what you think may be the cause of the bug

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```
Comment (Rostislav013) [inline .github/ISSUE_TEMPLATE/bug_report.md:11]: Shall we add here a template?
For ex: `Login page - name with 3 letters not accepted`
or for `Registration page - after clicking register happens nothing`
Like which components and what

**Review by MarinaFroes — COMMENTED** [review]
I'm kinda out of ideas for now. 😅

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+**❤️ Expected:**  
+
+## 🐛🔫 How to exterminate
+**😵 I already tried:**  
+**:shipit: suggested solution:**  
+
+## 🌴🌵 It lives in ❄️🌲
+describe the environment you found the bug in.  
+**💻 Device, OS:** [Samsung S10, Andoroid 7]  
+**🌐 Browser, version:** [Chrome, 55]
```
Comment (Rostislav013) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: Cool

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
```
[the remark this answers is not in the document]
Comment (Rostislav013) [reply .github/ISSUE_TEMPLATE/bug_report.md]: :shipit: - Possible solutions. I like that one

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+  
+*Ex. Why do I have  to see the same results every time I search? Why can't it remember that I don't want to play tennis?*   
+
+**🏆 My dream solution would be**  
+Describe the best possible scenario of this being implemented.  
+
+*Ex. User is never presented with the same activity twice.*  
+
+**:2nd_place_medal: But I'd also consider it solved if**    
+Describe an implementation that would fulfill the basic wish but might need more work in the future.
```
Comment (Rostislav013) [inline .github/ISSUE_TEMPLATE/feature_request.md:20]: Looks like 2nd place medal emoji doesn't work here

**Review by Rostislav013 — APPROVED** [review]

**Review by MarinaFroes — APPROVED** [review]
Seems good. 😃

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 7 above)
Comment (MikeVedsted) [reply .github/ISSUE_TEMPLATE/feature_request.md:20]: That's just for the Diff Viewer :) Works if you go to the preview.

**Pull request merged**